ui/src/treeDataProvider/AzureAccountTreeItemBase.ts

```diff
@@ -148,6 +149,14 @@ export abstract class AzureAccountTreeItemBase extends AzExtParentTreeItem imple
         return undefined;
     }
 
+    public compareChildrenImpl(item1: AzExtTreeItem, item2: AzExtTreeItem): number {
```

Comment:
Is this related?

Reply:
Yeah without it the items in the above screenshot would be sorted alphabetically:

- Create a Free Azure Account...
- Create New Local Project...
- Sign in to Azure...

I don't have a strong preference on the order _other than_ the two Azure ones should be next to each other. Which unfortunately is exactly what happens with alphabetical sorting.